packages/flutter/lib/src/rendering/viewport_offset.dart

```diff
@@ -209,5 +213,10 @@ class _FixedViewportOffset extends ViewportOffset {
   }
 
   @override
+  void jumpTo(double pixels) {
+    _pixels = pixels;
```

Comment:
This seems wrong. The whole point of a _FixedViewportOffset is that it's, well, fixed.